samples/excel/30-events/events-comment-event-handler.yaml

```diff
@@ -0,0 +1,195 @@
+order: 13
+id: excel-events-comment-event-handler
+name: Comment events
+description: 'Registers event handlers to listen for comment additions, changes, and deletions.'
+host: EXCEL
+api_set:
+    ExcelAPI: '1.12'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); 
+        $("#registerEventHandlers").click(() => tryCatch(registerEventHandlers)); 
+        $("#add-comment-to-selected-cell").click(() => tryCatch(addCommentToSelectedCell)); 
+        $("#add-comment-to-cell").click(() => tryCatch(addCommentToCell)); 
+        $("#edit-comment").click(() => tryCatch(editComment)); 
+        $("#delete-comment").click(() => tryCatch(deleteComment));
+
+        async function registerEventHandlers() {
+            await Excel.run(async (context) => {
+                const comments = context.workbook.worksheets.getActiveWorksheet().comments;
+
+                // Register the onAdded, onChanged, and onDeleted comment event handlers.
+                comments.onAdded.add(commentAdded);
+                comments.onChanged.add(commentChanged);
+                comments.onDeleted.add(commentDeleted);
+
+                await context.sync();
+
+                console.log("Added handlers for Comment onAdded, onChanged, and onDeleted events.");
+            });
+        }
+
+        async function commentAdded(event: Excel.CommentAddedEventArgs) {
+            await Excel.run(async (context) => {
+                // Retrieve the added comment.
+                let addedComment = context.workbook.comments.getItem(event.commentDetails[0].commentId);
+
+                // Load the added comment's data.
+                addedComment.load(["content", "authorName", "creationDate"]);
+
+                await context.sync();
+
+                // Print out the added comment's data.
+                console.log(`A comment was added:`);
+                console.log(`    ID: ${event.commentDetails[0].commentId}`);
```

Comment:
Instead of using the actual tab character, use `\t`. That will display consistently across browsers and IDEs. It also makes it so you can tell the difference between spaces and tabs at a quick glance.

Reply:
Per the yarn errors, I switched this file to use spaces instead of tabs. The console lines are all using an indentation of 4 spaces (I prefer 4-space to 2-space indentations, for readability). Using `\t` makes the script code less readable in my opinion, so I would rather use spaces. 

I have seen other samples use spaces for indentation similar to this implementation, so one of my goals was to be consistent with those. I haven't seen `\t` in any samples yet (but I also haven't seen all the samples). 

Are there issues with using spaces across browsers and IDEs?

Reply:
Using spaces is fine.